### ISSUE TITLE
[Schematics?] Hidden H1 doesn't get deployed

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13320]
**Describe the bug**
On the homepage, and when using a Screen Reader, there should be a hidden H1 available for the Elements Dialog. If deployment is done with yarn on local, the hidden H1 is there, however, when deploying to a demo server the H1 is missing, this might be a schematics issue. 

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**
1. Go to Homepage
2. Open VoiceOver for Mac or JAWS/NVDA for Windows
3. Open Rotor in VoiceOver and select "Headings" (ctrl+opt+U) or Elements Dialog and select "Headings" in JAWS (Ins+f3)or NVDA (Ins+f7)

**Expected behavior**
Hidden H1 should be available in all demo servers from version 4.0 and forward, regardless of deployment methodology.

**Screenshots**
![localhost 4](https://user-images.githubusercontent.com/15113219/127918099-263aa5aa-c923-46f0-b3fe-fb69a15d90ee.jpg)
![uiux server 4](https://user-images.githubusercontent.com/15113219/127918106-0354b45c-4e7c-4b66-8935-554f92acc9de.jpg)

**Desktop (please complete the following information):**
 - OS: BigSur 11.4
 - Browser Chrome
 - Version 92.0.4515.107

**Comment by saad-mo** [comment]
<img width="1625" alt="Screen Shot 2021-10-15 at 8 22 36 AM" src="https://user-images.githubusercontent.com/15215400/137493981-2f14a4c7-8285-4d0b-9fe3-3898e88efa5b.png">

The hidden `h1` renders fine on the homepage on the default dev server (https://40.76.109.9:9002) 

The [demo server](https://spartacus-demo.eastus.cloudapp.azure.com/electronics-spa/en/USD/) however does not render the `h1` and the main reason could be outdated sample data. 
- [This PR](https://github.tools.sap/cx-commerce/spartacussampledata/pull/167/files) merged the changes needed to render hidden `h1` on the homepage to `release/2005/next` 
- We need to ensure that the demo server has `release/2005/next` 

@giancorderoortiz @KateChuen any thoughts?

**Comment by saad-mo** [comment]
After latest sample data refresh on the default dev server, the homepage heading seems to render fine for VO

<img width="1521" alt="2021-10-22_10-49-18" src="https://user-images.githubusercontent.com/15215400/138485336-4a364411-177d-4c1d-bc49-fea6c59950e9.png">

**Comment by KateChuen** [comment]
Reopening because it's still an issue, depending on how we do the installtion. 
So, 
- installing with the develop repo of spartacus and using yarn, I see the H1
- installing using the script, i don't see the H1 (probably why we don't see it on the demo server either)
- installing using schematics, i don't see the H1

**Comment by developpeurweb** [comment]
From what I heard in a meeting with Team Blamed, the solution would be to include the homepage's H1 as a default in the installation script. Any doubts on how to fix this please reach out to @plabadie or @znikola.
The presence of the homepage's H1 is **needed for the audit.**

**Comment by Matejk00** [comment]
Fix ready for the review. Needs to be re-tested and approved by schematics owner.

**Comment by szczepaniak-michal** [comment]
I can see the h1 header on a newly generated app
<img width="804" alt="image" src="https://user-images.githubusercontent.com/97286147/159685092-cf9d3a51-d5d5-4f3b-a500-bae3b2fae444.png">

**Comment by KateChuen** [comment]
Tested using 
- develop branch of our project + yarn ✅
- develop branch of our project + installation script ✅

<img width="1434" alt="Screen Shot 2022-05-11 at 1 30 57 PM" src="https://user-images.githubusercontent.com/45409515/168083603-6b8389d9-f863-4040-a35a-c662e90ee9c4.png">

<img width="1215" alt="Screen Shot 2022-05-12 at 9 16 40 AM" src="https://user-images.githubusercontent.com/45409515/168083616-bdafe97c-71ec-46ce-b092-b9f0556930ba.png">


❗️❗️❗️❗️❗️ Was not able to test using new shell app + schematics yet, because it's not released yet.
